src/modules/FileLocksmith/FileLocksmithUI/Views/MainPage.xaml

```diff
@@ -101,23 +92,22 @@
             <Grid Visibility="{x:Bind ViewModel.IsLoading, Converter={StaticResource boolToVisibilityConverter}, Mode=OneWay}">
                 <ListView
                     x:Name="ProcessesListView"
-                    Grid.Row="1"
                     IncrementalLoadingThreshold="10"
                     ItemsSource="{x:Bind ViewModel.Processes}"
-                    SelectionMode="None">
+                    SelectionMode="None"
+                    Padding="12,0,12,12">
                     <ListView.ItemTemplate>
                         <DataTemplate x:DataType="interop:ProcessResult">
-                            <labs:SettingsExpander Margin="0,3,0,0">
+                            <labs:SettingsExpander Margin="0,0,0,4">
                                 <labs:SettingsExpander.Resources>
                                     <x:Double x:Key="SettingsCardWrapThreshold">0</x:Double>
                                 </labs:SettingsExpander.Resources>
                                 <labs:SettingsExpander.Header>
                                     <!--  We can't use the HeaderIcon because it only support a BitmapIcon, which only supports UriSource - not a direct BitmapImage  -->
-                                    <StackPanel Orientation="Horizontal">
+                                    <StackPanel Orientation="Horizontal" Spacing="8">
```

Comment:
Note for future: we might want to adopt the icon code DevHome is using, so we can just use the default `SettingsExpander` `Icon` / `Header` properties vs. nesting a custom header.